ShareX.ScreenCaptureLib/Shapes/Drawing/LineDrawingShape.cs

```diff
@@ -51,11 +51,6 @@ private void AdjustPoints(int centerPointCount)
                 newPoints[newPoints.Length - 1] = Points[Points.Length - 1];
             }
 
-            for (int i = 0; i < newPoints.Length; i++)
```

Comment:
Pretty sure removing this changes behavior. Instead of having an array filled with Points, you get one filled with null except the first and last entries.

Reply:
![image](https://user-images.githubusercontent.com/9832538/37573006-caeba00a-2b1b-11e8-9f62-5ffee0d09461.png)

newPoints is array of structs "System.Drawing.Point" https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/Point.cs,a041be61667d4c9a

Any element of an array can not be null. That expression (newPoints[i] == null) always return false. So, we can remove this cycle without changes in program logic.

Maybe we need to copy all the elements of the array points to new points?